CHANGELOG.md

```diff
@@ -8,6 +8,60 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.4.3] - 2023-04-12
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.4.2...v0.4.3)
+
+## Added
+* Release artifacts are now signed and include a Software Bill of Materials (SBOM) (#683)
+
+  The SBOM (Software Bill of Materials) is included in each Github release as using [Syft](https://github.com/anchore/syft) and is exported in [SBDX](https://spdx.dev/) format.
+
+  Developers will be able to verify the signature of the release artifacts with the following workflow(s):
+
+  ```shell
+  wget https://github.com/openfga/openfga/releases/download/<tag>/checksums.txt
+
+  cosign verify-blob \
+    --certificate-identity 'https://github.com/openfga/openfga/.github/workflows/release.yml@refs/tags/<tag>' \
+    --certificate-oidc-issuer 'https://token.actions.githubusercontent.com' \
+    --cert https://github.com/openfga/openfga/releases/download/<tag>/checksums.txt.pem \
+    --signature https://github.com/openfga/openfga/releases/download/<tag>/checksums.txt.sig \
+    ./checksums.txt
+  ```
+  if the `checksums.txt` validation succeeds, it means the checksums included in the release were not tampered with, so we can use it to verify the hashes of other files using the `sha256sum` utility. You can then download any file you want from the release, and verify it with, for example:
+
+  ```shell
+  wget https://github.com/openfga/openfga/releases/download/<tag>/openfga_<version>_linux_amd64.tar.gz.sbom
+  wget https://github.com/openfga/openfga/releases/download/<tag>/openfga_<version>_linux_amd64.tar.gz
+
+  sha256sum --ignore-missing -c checksums.txt
+  ```
+  And both should say "OK".
+
+  You can then inspect the .sbom file to see the entire dependency tree of the binary.
+
+  Developers can also verify the Docker image signature. Cosign actually embeds the signature in the image manifest, so we only need the public key used to sign it in order to verify its authenticity:
+  ```shell
+  cosign verify -key cosign.pub openfga/openfga:<tag>
+  ```
+
+* `openfga migrate` now accepts reading configuration from a config file and environment variables like the `openfga run` command (#655) - thanks @suttod!
+
+* The `--trace-service-name` command-line flag has been added to allow for customizing the service name in traces (#652) - thanks @jmiettinen
+
+## Fixed
+* Postgres and MySQL implementations have been fixed to avoid ordering queries by `ulid`. This noticably improves read query performance on larger OpenFGA stores (#677)
```

Comment:
```suggestion
* Postgres and MySQL implementations have been fixed to avoid ordering tuples by `ulid` when calling the Read API. This noticeably improves performance on larger OpenFGA stores (#677)
```

on the "noticeably": do we have proof?

Reply:
I would hope so. Otherwise how did we confirm the change?

@adriantam

Reply:
The Read API here is a bit misleading - it is check/list objects calling Read in datastore layer.

Reply:
@miparnisari regarding the change of wording, I'm going to change it to this instead

> Postgres and MySQL implementations have been fixed to avoid ordering relationship tuple queries by `ulid` when it is not needed. ...

The reason I'm favoring this instead of the suggestion is that it is not only relevant to the Read API. It also improves internal code paths for the Check, ListObjects, Read, and Expand API endpoints. So I wanted to simply mention that queries that return relationship tuples.

Reply:
I would not call it noticeably - it is just something that we notice is not optimal and change.

Reply:
OK, then I suggest

> * Postgres and MySQL implementations have been improved to avoid ordering relationship  tuples by `ulid` when it is not needed.

It's not a fix, it's an improvement :)

Reply:
> It's not a fix, it's an improvement :)
"Improvement" isn't a category in Keep a Changelog. Are you suggesting we put it under the "Changed" section.